src/pages/NewChatPage.js

```diff
@@ -179,10 +198,14 @@ class NewChatPage extends Component {
      * @param {Object} option
      */
     createChat(option) {
-        Report.fetchOrCreateChatReport([
-            this.props.session.email,
-            option.login,
-        ]);
+        let newChat = {};
+        const chat = this.getChatByParticipants([option.login]);
+        if (!chat) {
+            newChat = ReportUtils.buildOptimisticChatReport([option.login]);
```

Comment:
I'm not too sure on this one, but I think that we might need to add `OptionsListUtils.addSMSDomainIfPhoneNumber` in case the selected login is a phone number.

Reply:
I think you're right... that might even be an existing bug we have that was causing personal details to not return correctly. Updated both here and in createGroup